apiserver/params/secrets.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package params
+
+// CreateSecretArgs holds args for creating secrets.
+type CreateSecretArgs struct {
+	Args []CreateSecretArg `json:"args"`
+}
+
+// CreateSecretArg holds the args for creating a secret.
+type CreateSecretArg struct {
+	Type   string                 `json:"type"`
+	Path   string                 `json:"path"`
+	Scope  string                 `json:"scope"`
+	Params map[string]interface{} `json:"params"`
```

Comment:
Is there somewhere the valid keys/values for `Params` will be documented? Maybe a doc comment on each of these fields, with an example?

Reply:
params still evolving, i'll add a todo and some doc